api/management-model/build.gradle.kts

```diff
@@ -54,12 +58,26 @@ openApiGenerate {
   serverVariables = mapOf("basePath" to "api/v1")
 }
 
-listOf("sourcesJar", "compileJava").forEach { task ->
-  tasks.named(task) { dependsOn("openApiGenerate") }
-}
-
 sourceSets {
   main { java { srcDir(project.layout.buildDirectory.dir("generated/src/main/java")) } }
+  test { java { srcDir("src/test/java") } }
```

Comment:
Actually, I think even this change is not necessary - it's default in gradle. Only the new `testImplementation` deps are needed.